src/scripts/ci/travis.yml

```diff
@@ -88,6 +93,9 @@ matrix:
     - os: linux
       compiler: clang
       env: BUILD_MODE="cross-arm64"
+    # No docker build on osx
+    - os: osx
+      env: BUILD_MODE="docker-android"
```

Comment:
This should also exclude with gcc since NDK only uses Clang.

Reply:
In fact the docker build uses the clang binaries from the ndk, not the system one but maybe it is really necessary in travis case.

Reply:
The problem is right now there are two builds which are identical except for g++ vs clang. https://travis-ci.org/randombit/botan/jobs/554123055 https://travis-ci.org/randombit/botan/jobs/554123058

As you say it is kind of irrelevant which is used, since in either case we ignore CXX, but it would be best to have the Android build run as a Clang build so it is easy to see on the front page what compiler is being used, since it is *a* Clang that's doing the work.